server/services/nl.py

```diff
@@ -48,20 +48,134 @@
 MODEL_NAME = 'all-MiniLM-L6-v2'
 
 STOP_WORDS = {
```

Comment:
Can we move these constants and corresponding functions to one file in server/lib/nl_stop_words.py?

I would highly recommend to create a new folder: `server/lib/nl` and put current nl_*.py there too, so `lib/nl_data_spec.py` becomes `lib/nl/data_spec.py`

Reply:
I moved some constants to lib/nl/nl_constants.py and some processing to lib/nl/nl_utils.py